.npmignore

```diff
@@ -1,10 +1,17 @@
+external/shared/doc/
+external/shared/gen/
+external/shared/test/
+external/shared/tools/
 external/shared/samples/accounteditor/
 external/shared/samples/contactexplorer/
 external/shared/samples/fileexplorer/
 external/shared/samples/simplesync/
+external/shared/samples/simplesyncreact/
```

Comment:
It is. The big saving is the doc directory (which has the 500 pages PDF), it causes the tgz to go down from 25 to 21 MB. On the iOS side it's only 1.3 MB now.